Remotion/Web/Core/Doc/include/Configuration/ExecutionEngineConfiguration.xml

```diff
@@ -27,21 +27,14 @@
       <code>
 <![CDATA[
```

Comment:
I don't know what the preferred way of injecting a custom WxeLifetimeManagementSettings object would be

Reply:
Do a sample based on setting the ServiceLocator during startup:

```
      var defaultServiceLocator = DefaultServiceLocator.Create();
      var settings  = WxeLifetimeManagementSettings.Create(parameter1:10, parameter2: 5);
      defaultServiceLocator.RegisterSingle<IWxeLifetimeManagementSettings >(() => settings);
      ServiceLocator.SetLocatorProvider(() => defaultServiceLocator);
```
please fix the namings :)

Reply:
So the typical usecase of the provider is actually for prod and not just dev? I thought there was some other implementation. Thanks!

Reply:
Ah, yes. exactly. 

It's just the ServiceLocatorScope that's only for dev because we want to isolate the custom configs and the scope does this cleanup for us. The rest you see in the sample up there, that's startup code for the application and you chose the IoC infrastrucutre you want to use. Yes, it's possible and okay to set this up with Autofac instead. The sample doesn't need to detail those, it's enough to see how to configure the settings and the application developer knows how to wire up their IoC infrastructure.